mysite/scripts/snapshot_then_push.sh

```diff
@@ -30,16 +30,37 @@ cd "$PREFIX"
 
 ## Create a path to store the JSON in
 TEMPFILE="$(mktemp --suffix -snapshot.json.gz)"
+TEMPDB="$(mktemp --suffix -snapshot.db)"
+SNAPSHOT_SETTINGS_PATH="$(mktemp --suffix -snapshot_settings.py)"
```

Comment:
This will create a name that is not a valid Python module name. I'll be surprised if this works, but if it does, then that's cool

Sample output on my machine:

```
➜  ~  mktemp --suffix -snapshot_settings.py
/tmp/tmp.BflWujFDuM-snapshot_settings.py
```

Note that hyphens aren't typically easy to use in module names.